Makefile

```diff
@@ -107,6 +107,9 @@ include contrib/devtools/Makefile
 
 BUILD_TARGETS := build install
 
+tidy-all:
+	./scripts/go-mod-tidy-all.sh
```

Comment:
I included this from main to reduce headaches when messing with multiple go mods.